.dockerignore

```diff
@@ -1,7 +1,7 @@
 /static
 /node_modules
 /media
-*/ldap_config.py*
```

Comment:
The ldap config should be added by a volume and so not build in the image. As it may contains secrets...

Reply:
It's part of the app, not config files. It should nevers contains secrets, as there should be externalized.

Reply:
Need to be documented then

Reply:
done